src/test/kotlin/TestTacticalActionStrategy.kt

```diff
@@ -0,0 +1,50 @@
+import io.hsar.wh40k.combatsimulator.logic.TacticalActionStrategy
+import io.hsar.wh40k.combatsimulator.model.MapPosition
+import io.hsar.wh40k.combatsimulator.model.unit.BaseStat
+import org.junit.Test
+import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Not required

Reply:
As with the other test class, I am content to delete the decorator. I presume we need to keep the imports for the unit tests to work?

Reply:
You do not. The test runner class picks up all classes with methods annotated with Test, so it'll work just fine without this annotation.

Reply:
The imports are, however, useful for testing locally while working on the branch. I can delete them prior to merging if this is an issue?

Reply:
They shouldn't be required. What's happening locally?

Reply:
Stripped down to import org.junit.Test which is needed for the @Test decorator to be recognised.